health/health.c

```diff
@@ -1307,3 +1307,13 @@ void *health_main(void *ptr) {
     netdata_thread_cleanup_pop(1);
     return NULL;
 }
+
+void health_add_host_labels(void) {
+    DICTIONARY *labels = localhost->rrdlabels;
+
+    int is_ephemeral = appconfig_get_boolean(&netdata_config, CONFIG_SECTION_HEALTH, "is ephemeral", CONFIG_BOOLEAN_NO);
+    rrdlabels_add(labels, "_is_ephemeral", is_ephemeral ? "true" : "false", RRDLABEL_SRC_AUTO);
```

Comment:
Shouldn't it be `RRDLABEL_SRC_CONFIG`?

Reply:
Having it under `health` is questionable imo. Do you think it really fits @underhood?

Reply:
I have my doubts... reason i put it there is https://github.com/netdata/netdata/issues/13160#issuecomment-1256215166

Reply:
IMHO, it's fine to have it here. The primary thing is the caller to remain `rrdhost_load_auto_labels()`, which it does.

Reply:
thx, done